urgent-care/src/main/java/gov/va/api/health/urgentcare/service/controller/coverageeligibilityresponse/CoverageEligibilityResponseTransformer.java

```diff
@@ -65,27 +66,19 @@ CodeableConcept category(VceEligibilityInfo source) {
             .build());
   }
 
-  Reference coverage() {
-    return Reference.builder()
-        .identifier(
-            Identifier.builder()
-                .system("http://www.va.gov/identifiers/patients")
-                .id("PatientPlaceholder")
-                .build())
-        .build();
-  }
-
   private CoverageEligibilityResponse coverageEligibilityResponse(
       GetEeSummaryResponseTheRemix source) {
     return CoverageEligibilityResponse.builder()
         .resourceType("CoverageEligibilityResponse")
         .text(text())
+        .id(source.getIcn())
```

Comment:
Quoting @dtournour-va , "This shouldn't be here"

Reply:
We are moving forward. It is correct how it is.  Thanks for your feedback